### DISCUSSION TITLE
Add RoomTemperatureController as Climate entity?

### DISCUSSION THREAD

**Discussion by rernst** [kingsleyadam/local-abbfreeathome-hass#70] [Q&A] [answered]
Hi,

Thanks for the awesome work on this integration! I just installed it in a dev container and everything seems to work smoothly. 

The only device I am missing is a 'RoomTemperatureController'. [local-abbfreeathome](https://github.com/kingsleyadam/local-abbfreeathome) seems to have support for it since last week. So I was just wondering if someone (@derjoerg ?) is working on implementing this as a climate entity?

If not I might try to implement it my self (or help/test if needed), just have to find some time for it :).

Thanks again for the awesome work. 

Cheers,
Robert

**Comment by kingsleyadam** [comment]
My guess is @derjoerg was going to attempt to pull it in, I know it's a big one. @derjoerg , let us know.

I could add it but I have no way to test it so there may be some back and forth required.

**Reply by kingsleyadam** [reply]
Or @rernst, if you have some time and want to take a stab at it. I'm not sure how much you've worked with HA integrations.

**Reply by derjoerg** [reply]
Yep, it is on my todo :smiley:

I will create issues for all F@H devices and/or functionalities, which are already implemented in the lib but not yet in the HA-integration.

I have all open devices in my setup, so I'm able to test it also directly

**Reply by rernst** [reply]
Nice, let me know if you need any help with for example testing. I only have switches and a thermostat so can't do any other device testing.

I have very limited experience with home assistant integration development (just some minor bug fixing) so it would probably take me a while to get familiar. Do have plenty of python experience though.

**Comment by kingsleyadam** [comment]
I've created an issue for this platform addition.

**Reply by kingsleyadam** [reply]
https://github.com/kingsleyadam/local-abbfreeathome-hass/issues/71

**Comment by kingsleyadam** [comment, marked as answer]
Heads up @rernst , this was added in https://github.com/kingsleyadam/local-abbfreeathome-hass/releases/tag/0.13.0.

Please test and report back if you have any issues. For now I'll mark this as resolved.
